bin/aws-kinesis-agent-babysit

```diff
@@ -8,7 +8,7 @@ PIDFILE=/var/run/$DAEMON_NAME.pid
 SERVICE="service $DAEMON_NAME"
 
 function get_agent_pid() {
-  echo $(ps --ppid $(cat $PIDFILE 2>/dev/null) ho pid 2>/dev/null)
+  echo $(ps -p $(cat $PIDFILE 2>/dev/null) ho pid 2>/dev/null)
```

Comment:
Looks like Ubuntu behaves different from Redhat where the PID file actually contains the java process instead of the process id of start-aws-kinesis-agent